env.sh

```diff
@@ -1,14 +1,20 @@
 #!/bin/bash
 
-export FPGA_TOOL_PERF_DIR=`pwd`
+. ./env/conda/bin/activate
+source utils/environment.python.sh
 
 if [ -z "${VIVADO_SETTINGS}" ]; then
-    echo "ERROR: \$VIVADO_SETTINGS environmental variable needs to be set"
-    return 1
+    echo "WARNING: using default vivado settings"
+    VIVADO_SETTINGS=/opt/Xilinx/Vivado/2017.2/settings64.sh
 fi
 
 source ${VIVADO_SETTINGS}
```

Comment:
I'm pretty sure you don't want to source the vivado settings here. You should source then only *just* before you run a vivado command otherwise they mangle things like your libc.

Reply:
This requires changes in edalize, so that each time a vivado command is called, if the setting options is enabled. Changes are on a PR here: https://github.com/SymbiFlow/edalize/pull/31